src/main/java/com/minecolonies/core/entity/ai/workers/guard/RangerCombatAI.java

```diff
@@ -233,9 +235,7 @@ protected double getAttackDistance()
     @Override
     protected int getAttackDelay()
     {
-        // TODO: Maybe better for balancing to not increase damage and speed, looks odd and drains arrows/bow durability
-        final int attackDelay = RANGED_ATTACK_DELAY_BASE - (user.getCitizenData().getCitizenSkillHandler().getLevel(Skill.Adaptability));
-        return Math.max(attackDelay, PHYSICAL_ATTACK_DELAY_MIN * 2);
+        return RANGED_ATTACK_DELAY_BASE;
```

Comment:
Maybe we want to alter base slightly then?

Reply:
based 60 ticks seemed nice visually, so dont think we need to